packages/app-accounts/src/Editor.tsx

```diff
@@ -185,10 +185,10 @@ class Editor extends React.PureComponent<Props, State> {
     );
   }
 
-  onChangeAccount = (publicKey: Uint8Array): void => {
-    const current = publicKey && publicKey.length === 32
-      ? keyring.getPair(publicKey)
-      : null;
+  onChangeAccount = (ss58?: string): void => {
```

Comment:
Small nitpick (and yes, should have mentioned it before), it probably would make sense calling this `accountId` (which is basically the terminology used across Substrate/Polkadot)